barretenberg/cpp/src/barretenberg/bb/cli.cpp

```diff
@@ -551,6 +556,24 @@ int parse_and_run_cli_command(int argc, char* argv[])
     add_proof_path_option(avm_verify_command);
     add_vk_path_option(avm_verify_command);
 
+    /***************************************************************************************************************
```

Comment:
`msgpack run` is the feature described above that enables us to use bb as a server (but not currently on critical path, feature for the future)
`msgpack schema` is used by bb.js to generate the bindings - and I hope soon, we will also get some partners like zkpassport to help out and generate rust bindings, and they can move away from their bb-rs tooling entirely in favour of shipping the bb binary